<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolygonManager</class>
 <widget class="QMainWindow" name="PolygonManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инструмент выбора областей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цвет</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridButtons">
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonAddRegion">
        <property name="text">
         <string>Добавить регион интереса</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonAddPolygon">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Добавить полигон</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="buttonLoadregion">
        <property name="text">
         <string>Загрузить регион интереса</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="buttonRemoveRegion">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить регион интереса</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="buttonSaveRegion">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сохранить регион интереса</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
